src/main/java/org/cactoos/set/Sorted.java

```diff
@@ -35,11 +36,14 @@
  *
  * @param <T> Set type
  * @since 1.0.0
- * @todo #1292:30min This class should also implements SortedSet
- *  from the java collection framework by delegating to the
- *  wrapped set. Some tests must be added for it.
  */
-public final class Sorted<T> extends SetEnvelope<T> {
+@SuppressWarnings("PMD.OnlyOneConstructorShouldDoInitialization")
+public final class Sorted<T> extends SetEnvelope<T> implements SortedSet<T> {
```

Comment:
@vzurauskas can you introduce a `SortedSetEnvelope` (that extends `SetEnvelope`) responsible of delegating to a wrapped set and have `Sorted` only responsible of handling the `Iterable`-based constructor.

Reply:
@victornoel Yes, good idea.